CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 # Unreleased Changes
-## Suggested release: v4.2.1
+##### Suggested release: v4.3.0
 
 #### Bug fixes & Enhancements:
 - [#222](https://github.com/HewlettPackard/oneview-sdk-ruby/issues/222) Error listing the OS Deployment Plans from OneView
+- Added SCMB module and CLI command
```

Comment:
@fgbulsoni, I added the change under a "New Features" section, but there's no issue to attach it to, so still no link.